packages/flame_forge2d/example/lib/joints/constant_volume_joint.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:math';
+
+import 'package:flame/input.dart';
+import 'package:flame_forge2d/flame_forge2d.dart';
+import 'package:flame_forge2d_example/utils/ball.dart';
+import 'package:flame_forge2d_example/utils/walls.dart';
+
+class ConstantVolumeJointExample extends Forge2DGame with TapDetector {
+  @override
+  Future<void> onLoad() async {
+    super.onLoad();
+    addAll(createBoundaries(this));
+  }
+
+  @override
+  Future<void> onTapDown(TapDownInfo details) async {
+    super.onTapDown(details);
+    final tap = details.eventPosition.game;
+
+    const numPieces = 20;
+    const radius = 5.0;
+    final center = tap;
+    final balls = <Ball>[];
+
+    for (var i = 0; i < numPieces; i++) {
+      final xPos = radius * cos(2 * pi * (i / numPieces));
```

Comment:
```suggestion
      final x = radius * cos(2 * pi * (i / numPieces));
```
We don't allow abbreviations, you can just name the variables x and y here, it should be clear enough. :)